src/phrases.ts

```diff
@@ -48,3 +48,101 @@ export const matt = [
+  "Raven is Astounding!",
+  "Raven is Astonishing!",
+  "Raven is Remarkable!",
+  "Raven is Spectacular!",
+  "Raven is Incredible!",
+  "Raven is Extraordinary!",
+  "Raven is Sensational!",
+  "Raven is Marvellous!",
+  "Raven is a Genius!",
+  "Raven is Wonderful!",
+  "Raven is Majestic!",
+  "Raven is Mind-blowing!",
+  "Raven is Thrilling!",
+  "Raven is Fantastic!",
+  "Raven is Terrific!",
+  "Raven is Epic!",
+  "Raven is Pog!",
+  "Legend has it that when Raven looks you in the eyes, you see the universe.",
+  "Legend has it that when Raven walks in a room, music starts playing.",
+  "Legend has it that when Raven snaps his fingers, all bugs disappear.",
+  "Raven is soo Cool!",
+  "Raven is a Hero!",
+  "Raven is a Star!",
+  "Raven is Legendary!",
+  "Raven is 11/10!",
+  "Raven is Superb!",
+  "Raven is Wholesome!",
+  "Be like Raven, be Amazing",
+  "Raven does everything so well!",
+  "We all aspire to be Raven!",
+  "Raven (noun): An amazing person!",
+  "Raven (noun): An Astonishing person!",
+  "Raven (noun): A Remarkable person!",
+  "Raven (noun): A Spectacular person!",
+  "Raven (noun): An Incredible person!",
+  "Raven (noun): An Extraordinary person!",
+  "Raven (noun): A Sensational person!",
+  "Raven (noun): A Marvelous person!",
+  "Raven (noun): A Wonderful person!",
+  "Raven (noun): A Thrilling person!",
+  "Raven (noun): A Fantastic person!",
+  "Raven (noun): A Terrific person!",
+];
+export const luke = [
+  "We Love Luke!",
+  "Luke is Awesome!",
+  "Luke is the Best!",
+  "All hail Luke!",
+  "Luke is sooo Amazing!",
+  "Luke is Astounding!",
+  "Luke is Astonishing!",
+  "Luke is Remarkable!",
+  "Luke is Spectacular!",
+  "Luke is Incredible!",
+  "Luke is Extraordinary!",
+  "Luke is Sensational!",
+  "Luke is Marvellous!",
+  "Luke is a Genius!",
+  "Luke is Wonderful!",
+  "Luke is Majestic!",
+  "Luke is Mind-blowing!",
+  "Luke is Thrilling!",
+  "Luke is Fantastic!",
+  "Luke is Terrific!",
+  "Luke is Epic!",
+  "Luke is Pog!",
+  "Legend has it that when Luke looks you in the eyes, you see the universe.",
+  "Legend has it that when Luke walks in a room, music starts playing.",
+  "Legend has it that when Luke snaps his fingers, all bugs disappear.",
+  "Luke is soo Cool!",
+  "Luke is a Hero!",
+  "Luke is a Star!",
+  "Luke is Legendary!",
+  "Luke is 11/10!",
+  "Luke is Superb!",
+  "Luke is Wholesome!",
+  "Be like Luke, be Amazing",
+  "Luke does everything so well!",
+  "We all aspire to be Luke!",
+  "Luke (noun): An amazing person!",
+  "Luke (noun): An Astonishing person!",
+  "Luke (noun): A Remarkable person!",
+  "Luke (noun): A Spectacular person!",
+  "Luke (noun): An Incredible person!",
+  "Luke (noun): An Extraordinary person!",
+  "Luke (noun): A Sensational person!",
+  "Luke (noun): A Marvelous person!",
+  "Luke (noun): A Wonderful person!",
+  "Luke (noun): A Thrilling person!",
+  "Luke (noun): A Fantastic person!",
+  "Luke (noun): A Terrific person!",
+];
```

Comment:
```suggestion
];

```
New line at the end of the file should fix the lint issue.